Ejercicios semana 1

sheifg opened the pull request:
### Resolución del ejercicio de la clase 2 

- Tras hacer _fork_ del repositorio de la semana 1, he clonado mi repositorio en mi HOME. Para ello he copiado el link del mismo y con el comando _git clone "link"_ en VS Code, ya he clonado el archivo en mi ordenador
- Con el archivo en mi ordenador he creado un documento en la carpeta de _github- profiles_ con mi usuario de Discord (Sheila_FG-7067). Después he incluido el archivo de Markdown de README de mi perfil de GitHub que he creado en el ejercicio de la primera clase, añadiendo al final el link de mi perfil
- Una vez guardado el archivo, para hacer commit en el servidor he seguido los siguientes pasos:
    - Comando _git add Sheila_FG-7067_, para añadir el archivo a stage o con _git add *_, añadiendo todo
    - Comando _git commit -m "First commit"_, con esto he creado el commit de forma local en mi ordenador
    - Comando _git push_, de esta manera estoy enviandolo al servidor, en este caso GitHub
- Como último paso estoy creando el Pull Request

misterdan100 commented:
Hola, me gustó mucho el ejercicio que realizaste.

Solamente recomendaría dos cosas.
- Renombrar el archivo con la extensión .md para que puede ser visualizado directamente en GitHub en versión markdown y se sepa que se trata de tal tecnología.
- Hay que darle un nombre más descriptivo a los commits para poder ver solo con el título del commit de que se trata.

Felicidades por tu avance 👏🏼

misterdan100 reviewed (changes requested):
Hola, me gustó mucho el ejercicio que realizaste.

Solamente recomendaría dos cosas.
- Renombrar el archivo con la extensión .md para que puede ser visualizado directamente en GitHub en versión markdown y se sepa que se trata de tal tecnología.
- Hay que darle un nombre más descriptivo a los commits para poder ver solo con el título del commit de que se trata.

Felicidades por tu avance 👏🏼

sheifg commented:
@misterdan100 muchas gracias por las recomendaciones Daniel :)
He realizado los cambios que me has sugerido y también lo tendré en cuenta para los siguientes ejercicios.

sheifg commented:
### Resolution of exercise 2
* Inside the github-profiles folder I have created a folder with my Discord username _Sheila_FG-7067_
* With the folder created I have renamed the Markdown file of my GitHub created in the exercise 1 with _README.md_ and moved it there
* In order to continue I have made an _index.html_ based on _README.md_
* To conclude I have sent a commit of the file _index.html_ and named it with _Creation of HTML file based on the Markdown file from my profile_

sheifg commented:
### Resolution of class 4 exercise
- Check the tutorial "Web Design in 4 minutes" of Jeremy Thomas
- Review what I learned during the class and decide how I wanted to improve the appearance of my index.html through CSS
- Add _<div>_ to organise into sections and include _id_ and _class_ selectors to select the HTML element(s) I want to style
- Create the file style.css and style the _index.hmtl_: body, content, tittles, paragraphs, tables, links... 
- Search for CSS tutorials to do everything I want to do
- Once everything was done, sent the commit _Adding CCS to style index.html_

cristinacdm reviewed (approved):
Todo parece bien, good job!👍🏻

sheifg commented:
@cristinacdm muchas gracias por revisar mi PR :)

Byron2016 commented on github-profiles/Sheila_FG-7067/README.md:
@undefined-academy :+1: This PR looks great - it's ready to merge! :shipit:

Hola, @Sheila_FG

- [x] Nombre de carpeta: Correcto
- [x] README.md: me pareció que  bien, no importante pero que algo dijo Guillermo es que para tema de legibilidad el raw deberían las tablas estar alineadas, pero no es algo necesario en realidad.

Byron2016 commented on github-profiles/Sheila_FG-7067/github-mark.png:
@undefined-academy :+1: This PR looks great - it's ready to merge! :shipit:

Hola, @Sheila_FG

- [x] .png: podría estar en una carpeta img, pero sobre esto Guillermo no indicó nada.

Byron2016 reviewed (commented):
@undefined-academy :+1: This PR looks great - it's ready to merge! :shipit:

Hola, @Sheila_FG

- [x] En cuanto a los commits de pronto completar la idea, por ejemplo agregas extensión a archivo pero no pones a que archivo

Byron2016 reviewed (changes requested):
Hola, @Sheila_FG

- [x] Archivo Sheila_FG-7067.md: **No** debería estar directamente en carpeta Github-profiles
- [x] .png: podría estar en una carpeta img, pero sobre esto Guillermo no indicó nada.

sheifg commented:
@Byron2016 Muchas gracias por tus comentarios y he realizado los cambios que me sugeriste. Respecto al archivo Sheila_FG-7067.md lo había sustituido por el README.md que aparece, pero creo que por un error en cuanto a ubicación en carpetas al hacer push, no se actualizó o sincronizó.

Byron2016 reviewed (approved):
@undefined-academy :+1: This PR looks great - it's ready to merge! :shipit:
